Android/build.gradle

```diff
@@ -1,7 +1,7 @@
-dependencyLocking {
-    lockAllConfigurations()
```

Comment:
We still need to lock the dependencies, this might lead to a large amount of changes.

Reply:
hi @jyyi1 please help to re-review my PR, I was adding two more commits:
- https://github.com/Jigsaw-Code/Intra/pull/524/commits/a35b414de5d68d2aa1069dff828186000bff6dc7 
- https://github.com/Jigsaw-Code/Intra/pull/524/commits/cfe5ca137f86907ab6d2290d30d14dcfff7c6cc6

let me know if you have any feedback